test/e2e/tests/custom-token-send-transfer.spec.js

```diff
@@ -0,0 +1,426 @@
+const {
+  convertToHexValue,
+  withFixtures,
+  getWindowHandles,
+} = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+const { SMART_CONTRACTS } = require('../seeder/smart-contracts');
+
+describe('Custom Token', function () {
+  const smartContract = SMART_CONTRACTS.HST;
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: convertToHexValue(25000000000000000000),
+      },
+    ],
+  };
+  it('can be sent from inside MetaMask', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: new FixtureBuilder()
+          .withPermissionControllerConnectedToTestDapp()
+          .build(),
+        ganacheOptions,
+        smartContract,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver, contractRegistry }) => {
+        const contractAddress = await contractRegistry.getContractAddress(
+          smartContract,
+        );
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+        // create token
+        await driver.openNewPage(
+          `http://127.0.0.1:8080/?contract=${contractAddress}`,
+        );
+        let windowHandles = await driver.getAllWindowHandles();
+        const extension = windowHandles[0];
+        await driver.findClickableElement('#deployButton');
+        await driver.clickElement({
+          text: 'Add Token to Wallet',
+          tag: 'button',
+        });
+
+        await driver.waitUntilXWindowHandles(3);
+        windowHandles = await driver.getAllWindowHandles();
+        // switch to popup and add token from dapp
+        await driver.switchToWindowWithTitle(
+          'MetaMask Notification',
+          windowHandles,
+        );
+        await driver.clickElement({ text: 'Add token', tag: 'button' });
+        await driver.waitUntilXWindowHandles(2);
+        await driver.switchToWindow(extension);
+
+        const asset = await driver.waitForSelector({
+          css: '.list-item',
+          text: '10 TST',
+        });
+        asset.click();
+        // send token from extension
+        await driver.waitForSelector('[data-testid="eth-overview-send"]');
+        await driver.clickElement('[data-testid="eth-overview-send"]');
+
+        await driver.fill(
+          'input[placeholder="Search, public address (0x), or ENS"]',
+          '0x2f318C334780961FB129D2a6c30D0763d9a5C970',
+        );
+        await driver.fill('.unit-input__input', '1');
+        // continue to next screen
+        await driver.clickElement('[data-testid="page-container-footer-next"]');
+        // added wait for selector since it needs some time to load this page
+        // otherwise would have to use delays
+        await driver.waitForSelector({ text: 'Edit', tag: 'button' });
```

Comment:
we should be able to remove `await driver.waitForSelector({ text: 'Edit', tag: 'button' });`

Reply:
Done